db/db.js

```diff
@@ -0,0 +1,16 @@
+const { Sequelize } = require("sequelize");
+const sequelize = new Sequelize("sqlite::memory:");
+module.exports = sequelize;
+
+const sequelize = new Sequelize({
+  dialect: "postgres",
+  database: process.env.DATABASE,
+  username: process.env.DATABASE_USER,
+  password: process.env.DATABASE_PASSWORD,
+  host: "localhost",
```

Comment:
This is also a good implementation of defining the object. 
The `host` directive will be useful when deploying in containers.